src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/ActionType.kt

```diff
@@ -41,7 +41,8 @@ data class ActionType<out T>(
 @JvmField val EDIT_ISSUE_SUBMIT = ActionType("Edit Issue") { IssueObservation(it) }
 @JvmField val PROJECT_SUMMARY = ActionType("Project Summary") { Unit }
 @JvmField val SEARCH_WITH_JQL = ActionType("Search with JQL") { SearchJqlObservation(it) }
-@JvmField val SEARCH_WITH_JQL_A_STAR = ActionType("Search with JQL a*") { SearchJqlObservation(it) }
+@JvmField val SEARCH_WITH_JQL_WITH_WILDCARD = ActionType("Search with JQL a*") { SearchJqlObservation(it) }
+@JvmField val SEARCH_JQL_SUBSET = ActionType("Search with JQL except a* and reporter was admin") { SearchJqlObservation(it) }
```

Comment:
I don't like the current names/labels for actions. It makes it so non-open-closed. It will be hard to extend it in the future, in case we'd like to add more JQLs.
My ideas (I'm not good with names and I think @sebapawlak or @dagguh  could help here):
`SEARCH_WITH_JQL_WITH_WILDCARD`/`Wildcard searches`  -> all the heavy searches with wildcard used
`SEARCH_JQL_SIMPLE`/`Simple searches` -> basic term searches on the issue index
`SEARCH_JQL_CHANGELOG`/`Changelog searches` -> queries that use changelog index. The queries are slow not because of complexity, but because it's the largest index.

Reply:
I fucking love it.